src/sql/azdata.proposed.d.ts

```diff
@@ -8,6 +8,23 @@
 import * as vscode from 'vscode';
 
 declare module 'azdata' {
+
+	export namespace env {
+		/**
+		 * Well-known app quality values
+		 */
+		export enum AppQuality {
```

Comment:
IIRC, there is also a rc1 quality, right? as far as this new property is concerned, it should be treated as `stable`. please make sure it is the case.